src/errors/ajax.ts

```diff
@@ -28,7 +28,7 @@ class AjaxErrors extends Base {
     const xhrSend = XMLHttpRequest.prototype.send;
     const xhrEvent = (event: any) => {
       try {
-        if (event && event.currentTarget && event.currentTarget.status !== 200) {
+        if (event && event.currentTarget && event.currentTarget.status !== 200 && event.currentTarget.status !== 304) {
```

Comment:
Most (if not all) of the other agents use a condition `status >= 400` to determine the status is `error` or not, does it apply to this agent too?

Reply:
All right

Reply:
Yes, `>=400` is correct check. 202 is also correct, 302 is only redirect.